docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,229 @@
+```console
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+This is the full object path that we will be using in examples below.
+
+**Tokens and JWT:**
+Some namespace prefixes are public, like `/ospool/uc-shared/public`, while others are protected (i.e. they require authorization). Objects in public namespaces can be downloaded by anybody, but downloading objects from protected namespaces requires you prove to the origin supporting that namespace that you are allowed to access the object. In Pelican, this is done using signed JSON Web Tokens, or *JWT*s for short. In many cases, these tokens can be generated automatically.
+
+## The Different Pelican URL Schemes
+When running client commands with a request URL, there are three different URL schemes a user can use: `pelican` and `osdf`. This page uses examples with the `pelican://` URL scheme. Below is a description of the three URL schemes and what to expect when they are used:
+
+### The `pelican://` URL Scheme:
+URLs that being with `pelican://` allow you to interact with objects from any federation simply by knowing the hostname of the federation's discovery url and the name of that object in that federation. These URLs take the form:
+```bash
+pelican://<federation discovery URL>/<path/to/object>
+```
+When a Pelican client encounters a Pelican URL, it uses the hostname in the federation discovery portion of the URL to find other services in the specified federation that will help the client discover the object's location
+
+>**Note**: Federation discovery URLs can also be specified with the `-f` flag rather than the hostname. Just be sure not to include the URL in both the `-f` flag and in the hostname of the object path, as this will lead to errors.
+
+>**Note:** Federation discovery URLs cannot contain a path component (e.g. https://my-federation.com/path/component). Clients that encounter a discovery URL with a path component will incorrectly assume the path is part of the namespace prefix.
+
+### The `osdf:///` URL Scheme:
+When using the `osdf:///` scheme, pelican will load in the federation metadata automatically with the Open Science Data Federation (OSDF)  defaults. This scheme should only be used if your federation or data can be accessed through the OSDF. When using this scheme, you should not specify the federation URL at all, as Pelican will handle this automatically. For example:
+```bash
+pelican object get osdf:///<namespace-prefix></path/to/file> <local/path/to/file>
+```
+>This scheme has three slashes (`///`) after the `osdf` because the hostname is left empty to be automatically populated therefore, just start the URL with the namespace prefix. Pelican also supports the `osdf://` scheme (with two slashes) in case the user forgets to pass the third slash, but this is not recommended.
```

Comment:
```suggestion
>This scheme has three slashes (`///`) after the `osdf` because the hostname is left empty to be automatically populated therefore, just start the URL with the namespace prefix. Pelican currently recognizes the `osdf://` scheme (with two slashes) in case the user forgets to pass the third slash, but this is not recommended.
```